clientd3d/d3drender_world.h

```diff
@@ -0,0 +1,21 @@
+// Meridian 59, Copyright 1994-2024 Andrew Kirmse and Chris Kirmse.
+// All rights reserved.
+//
+// This software is distributed under a license that is described in
+// the LICENSE file that accompanies it.
+//
+// Meridian is a registered trademark.
+//
+// The render world interface is responsible for rendering the 3d game world
+// (including buildings, scenery, roads and so on). The 3d world is always drawn 
```

Comment:
Buildings and roads have no representation in data.  Maybe say that it's responsible for drawing the elements of the BSP tree?